internal/cli/kraft/cloud/utils/print.go

```diff
@@ -611,15 +611,22 @@ func PrintServiceGroups(ctx context.Context, format string, resp kcclient.Servic
 
 // An internal utility method for printing a bar based on the provided progress
 // and max values and the width of the bar.
-func printBar(progress, max int) string {
+func printBar(cs *iostreams.ColorScheme, progress, max int) string {
 	width := 36
 
 	var ret strings.Builder
 
 	percent := math.Floor(float64(progress) / float64(max) * float64(width))
 
-	ret.WriteString(strings.Repeat("█", int(percent)))
-	ret.WriteString(strings.Repeat("░", width-int(percent)))
+	color := "green"
+	if percent > 30 {
```

Comment:
Isn't `30` a bit small

I would do:
1. [0-60) green
2. [60-80) yellow
3. [80-100] red

Reply:
It's out of 36, not 100. 30 represents ~83%